src/dlr_relayvm.cc

```diff
@@ -237,6 +232,27 @@ void RelayVMModel::SetInput(const char* name, const int64_t* shape, const void*
   inputs_[index] = input_arr;
 }
 
+void RelayVMModel::SetInput(const char* name, DLTensor* tensor) {
```

Comment:
Why is this showing up in diff? Did it get duplicated?

Reply:
this is a new API. Its different from the existing one:
https://github.com/neo-ai/neo-ai-dlr/blob/3571813712412b38ef5b086fcc6cdf088f7b613a/src/dlr_relayvm.cc#L213

Reply:
I see, thanks!